### DISCUSSION TITLE
戰爭中的 AI：烏克蘭用臉部辨識俄羅斯士兵，但這只是小兒科

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#388] [科技新知]
現今的AI技術已經大幅應用在戰場上。此種用AI辨識敵方士兵的方式，讓全世界都知道這個人在攻擊烏克蘭
「據《Wired》報導，烏克蘭方網路偵探使用臉部辨識技術，僅透過螢幕截圖就能5分鐘辨識俄羅斯士兵身分。」」
再加上網路社群媒體的發達，此方式會大幅增加此人的曝光。
這種剝奪隱私權制衡戰爭問題的方式，值得更多的討論

資料來源：https://technews.tw/2022/03/21/ai-in-war/

**Comment by lx3211** [comment]
可是感覺這滿不公平的，也不是士兵們說不上戰場就能不上戰場的，道德譴責他們也沒什麼用

**Comment by NcuJerryKao** [comment]
感覺事也威攝力的 讓他們知道 我知道你是誰 就像知道搶匪身分 知道強盜車牌號碼一樣

**Comment by limitx0** [comment]
感覺用臉部辨識知道士兵的身分也沒什麼用
